fix test

giolandicho opened the pull request:
too many args

giolandicho committed:
fix test

too many args

giolandicho committed:
Create RewardScheduleV3.test.js

giolandicho committed:
Delete RewardScheduleV3.test.js

giolandicho committed:
fix bug and test for RewardScheduleV3

giolandicho committed:
Merge branch 'fix/points_system' of https://github.com/giolandicho/fatex-dex-protocol into fix/points_system

giolandicho committed:
fix how fateperblock is calculated

giolandicho committed:
add test for FatePerBlock queries that span the same week

coreycaplan3 reviewed (commented):
Good start but needs to add integration tests.

coreycaplan3 reviewed (commented):
A good start adding the new test case and changing the numbers to work with the updated schedule. Next steps should be taken to add unit tests that assert proper withdrawal fees and reward fees are applied to different deposit lengths. Integration tests should be added after that replicate the production environment via mainnet forking.

giolandicho committed:
deposit, withdraw tests

checks if userMembershipInfo is updated after deposit and withdraw functions are called

giolandicho committed:
test pendingLockedFate and pendingUnlockedFate

giolandicho committed:
test withdraw fees
